pdns/dnsmessage.proto

```diff
@@ -45,6 +45,28 @@ message PBDNSMessage {
     Truncate= 5;                                // https://tools.ietf.org/html/draft-vixie-dns-rpz-04 3.5
     Custom = 6;                                 // https://tools.ietf.org/html/draft-vixie-dns-rpz-04 3.6
   }
+  enum VState {
+    Indeterminate = 1;
+    Insecure = 2;
+    Secure = 3;
+    NTA = 4;
```

Comment:
The final validation state should never be `NTA` or `TA`, I'd consider that a bug.